pkg/failover/failover.go

```diff
@@ -77,6 +78,16 @@ func (f *Failover) Run(ctx context.Context, deferCtx context.Context) error {
 	)
 }
 
+func (f *Failover) HealthCheckKeepers(ctx context.Context) error {
```

Comment:
What if we have no master? hahaha

Reply:
As discussed this is updated to include the master now, and is changed tot CheckClusterHealth.